src/style.css

```diff
@@ -1,3 +1,68 @@
+/* CSS variables */
+:root {
+  --active-text-color: #00acdc;
+}
+
 body {
+  font-family: Roboto, Arial, sans-serif;
+  margin: 0;
+}
+
+.logo-header {
+  line-height: 0;
+}
+
+.container {
+  display: flex;
+  margin: 0 auto;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 1200px;
+  min-width: 780px;
```

Comment:
Not hardcoded or using static dimensions.
It is better to set more flexible sizes, for example (max-width: 100%)